Formula/ksync.rb

```diff
@@ -2,8 +2,9 @@ class Ksync < Formula
   desc "Sync files between your local system and a kubernetes cluster"
   homepage "https://ksync.github.io/ksync/"
   url "https://github.com/ksync/ksync.git",
-      tag:      "0.4.6",
-      revision: "bfb445b179a0405ab4a01b999010010406f425b7"
+      tag:      "0.4.7-hotfix",
+      revision: "14ec9e24670b90ee45d4571984e58d3bff02c50e"
+  version "0.4.7" # remove when tag is a proper version
```

Comment:
Oops, maybe this isn't needed. Couldn't hurt, I guess?

Reply:
`0.4.7` could also removed upstream if that would help avoid this case. I don't want to disrupt anyone's current workflow.

Reply:
No, there's no need for you to do that on our account. Thanks for the offer though!